### ISSUE TITLE
[Suggestion] Sign Text Alignment & Scaling

### ISSUE THREAD

**Issue by Tim3Game** [DonBruce64/MinecraftTransportSimulator#507]
## Sign Text Alignment
Pack creators would have the ability to align the text in different **textLines**. Would look much nicer and more realistic.

*this type of alignment:*
<img src="https://user-images.githubusercontent.com/34278596/90652737-b02fe500-e23e-11ea-9a0a-5bdd13ca8fbb.png" width="15%">

**sign json file:**
```json
"textLines" : [
      {
        "align": "left|center|right"
      },
]
```
For instance here you can see, some of the text is aligned left and some right:
<img src="https://user-images.githubusercontent.com/34278596/90652961-eb321880-e23e-11ea-8739-d4b17123fd25.png" width="50%">

## Sign Text Scaling
This would scale the text to fit the defined width, if activated, in the json file. With this, signs wouldn’t need multiple textLines for different sizes, but would simply just set the width and the text would automatically scale to size. And if the author wouldn’t need this behavior on specific lines, he would simply disable it.
![obrázok](https://user-images.githubusercontent.com/34278596/90653514-8fb45a80-e23f-11ea-8acc-9892aca72510.png)

**sign json file:**
```json
"textLines" : [
      {
        "scaling": "true|false",
        "scaleWidth": 5,
      },
]
``

**Comment by DonBruce64** [comment]
The only tricky thing to do on this would be the right-aligned text.  MC doesn't allow for such alignment with its standard system, so you'd have to calculate the width of the text and offset the render by that amount.  Still, it's do-able, so will do my best to get it working.

**Comment by DonBruce64** [comment]
Implemented in V19.0.0.
